### PR TITLE
[Ansible] Add apiserver-count as apiserver option

### PR DESCRIPTION
**Pull request by galexrt**
Fix the `kube_apiserver_options` variable usage in the template
Removed duplicate variables from the master and node role defaults

/cc @xialonglee @axhoffmann For even better multi master clusters ;)

### CODE REVIEW

File: ansible/roles/node/defaults/main.yml [hunk 1]
```diff
@@ -15,9 +15,6 @@ kube_node_rpms:
   - kubernetes-client
   - kubernetes-node
 
-kubelet_options: []
-#  - "--host-network-sources=*"
-#  - "--pod-infra-container-image=gcr.io/google_containers/pause:2.0"
 kube_proxy_options: []
```
Comment (ingvagabund) [inline ansible/roles/node/defaults/main.yml]: This one can be removed as well

**Comment by galexrt** [comment]
@ingvagabund I removed the other duplicate variable.

**Comment by xialonglee** [comment]
It's good!

**Comment by ingvagabund** [comment]
@redhat-k8s-bot test please

**Pull request merged**